lib/plugins/package/lib/packageService.js

```diff
@@ -52,9 +52,12 @@ module.exports = {
   packageAll() {
     const exclude = this.getExcludes();
     const include = this.getIncludes();
-    const zipFileName = this.provider.naming.getServiceArtifactName();
+    const zipFileName = `${this.serverless.service.service}.zip`;
```

Comment:
Thanks for the explanation. Yes, I agree. A provider independent naming module would increase tech-debt and might be a little bit overkill for now. But something we should keep in mind if we see other patterns where we could utilize this again.